.github/PULL_REQUEST_TEMPLATE/feature.md

```diff
@@ -0,0 +1,33 @@
+# ✨ Feature Implementation
+
+**Related Issue:**
+> Implements Story #
+> Fixes #
+
+### 📝 Description
+
+<!-- (Briefly describe the feature, e.g., "Implemented the Virtual Machine instruction set") -->
+
+<!-- Decisions made:
+<!-- (e.g., "Used a flat list for instructions as per Part 2 requirements") -->
+
+### 🛠️ Type of Change
+
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] `0. layer: story` implementation
+- [ ] `0. layer: task` implementation
+
+### 🧪 Testing Policy
+
+- [ ] Unit Tests added in (path/to/test)
+- [ ] Integration Tests added (e.g., new `.scm` files)
+- [ ] Coverage check passed
+
+### ✅ Checklist (Definition of Done)
+
+- [ ] My code follows the style guidelines of this project (clean code)
+- [ ] I have performed a self-review of my code
+- [ ] `make` compiles without error
+- [ ] `make clean` and `make fclean` work correctly
```

Comment:
Why is this a necessary step for the pr to be considered as done? If `make clean` is broken, there is bugfix needed, be it should not block the whole project?